docker/exporter/export_runner.py

```diff
@@ -74,6 +78,39 @@ def spawn_ecosystem_exporter(work_dir: str, bucket: str, eco: str):
     logging.error('Export of %s failed with Exit Code: %d', eco, return_code)
 
 
+def aggregate_all_vulnerabilities(work_dir: str, export_bucket: str):
+  """
+  Aggregates vulnerability records from each ecosystem into a single zip
+  file and uploads it to the export bucket.
+  """
+  zip_file_name = 'all.zip'
+  output_zip = os.path.join(work_dir, zip_file_name)
+  all_vulns = {}
+
+  for ecosystem_dir, _, files in os.walk(work_dir):
+    if ecosystem_dir == work_dir:
+      continue
+    for vuln_filename in files:
+      if not vuln_filename.endswith('.json'):
+        continue
+      all_vulns[vuln_filename] = os.path.join(ecosystem_dir, vuln_filename)
+
+  with z.ZipFile(output_zip, 'a') as all_zip:
```

Comment:
Apparently python's zip library uses a lot of memory (which might be why we run out of memory in our current exporter), so this might be a problem. No action required for now, though if this ends up using too much memory we can replace this zip library with a streaming one.